sample/composer.json

```diff
@@ -12,10 +12,10 @@
         "classmap": [
             "models",
             "schemas"
-        ]
+        ],
+      "psr-4": {
+        "Neomerx\\JsonApi\\": "../src/"
+      }
     },
-    "minimum-stability": "dev",
```

Comment:
that's a composer file for the sample app. It does need to load models and schemas
sorry but this change is wrong